account_banking_payment_export/wizard/payment_order_create.py

```diff
@@ -45,9 +46,33 @@ def default_get(self, field_list):
     def extend_payment_order_domain(self, payment_order, domain):
         if payment_order.payment_order_type == 'payment':
             domain += [('account_id.type', 'in', ('payable', 'receivable')),
-                       ('amount_to_pay', '>', 0)]
+                       ('credit', '>', 0)]
```

Comment:
Sorry, it was in 7.0 (https://github.com/odoo/odoo/blob/7.0/addons/account_payment/account_move_line.py#L29), but it has been removed in 8.0.

Are you taken into account currencies and so on?